test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -370,6 +370,8 @@ $redirectTests = @(
 
 Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
     BeforeAll {
+        $oldProgress = $ProgressPreference
```

Comment:
> I was running the tests locally and the numerous progress bars being rendered created a different problem (hitting a debug assert)

Funny, but I recently had the same assert issue with tests for PSGetV3 and also suppressed progress bar in tests.  It looks like that assert is incorrect and may affect other tests.  Can you create an issue and/or PR to remove the assert?

Reply:
Yeah, I'll take care of it in a separate PR

Reply:
https://github.com/PowerShell/PowerShell/pull/16588